libs/platform/src/lib/components/dynamic-page/dynamic-page-content/dynamic-page-tabbed-content.component.ts

```diff
@@ -0,0 +1,155 @@
+import { CdkScrollable, ScrollDispatcher } from '@angular/cdk/scrolling';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ElementRef,
+    forwardRef,
+    NgZone,
+    Renderer2,
+    ViewChild,
+    Input
+} from '@angular/core';
+
+import { DYNAMIC_PAGE_CHILD_TOKEN, BACKGROUND_TYPE, RESPONSIVE_SIZE, CLASS_NAME } from '../constants';
+import { DynamicPageService } from '../dynamic-page.service';
+import { addClassNameToElement } from '../utils';
+
+@Component({
+    selector: 'fdp-dynamic-page-tabbed-content',
+    template: '<div class="fd-dynamic-page__content"><ng-content></ng-content></div>',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [
+        {
+            provide: DYNAMIC_PAGE_CHILD_TOKEN,
+            useExisting: forwardRef(() => DynamicPageTabbedContentComponent)
+        }
+    ]
+})
+export class DynamicPageTabbedContentComponent extends CdkScrollable {
+    /**
+     * sets background for content to List, Transparent or Solid background color.
+     * Default is `solid`.
+     */
+    @Input()
+    set background(backgroundType: BACKGROUND_TYPE) {
+        if (backgroundType) {
+            this._background = backgroundType;
+            this._setBackgroundStyles(backgroundType);
+        }
+    }
+
+    get background(): BACKGROUND_TYPE {
+        return this._background;
+    }
+
+    /**
+     * sets size which in turn adds corresponding padding for the size type.
+     * size can be `small`, `medium`, `large`, or `extra-large`.
+     */
+    @Input()
+    set size(sizeType: RESPONSIVE_SIZE) {
+        if (sizeType) {
+            this._size = sizeType;
+            this._setSize(sizeType);
+        }
+    }
+
+    get size(): RESPONSIVE_SIZE {
+        return this._size;
+    }
+
+    @ViewChild(CdkScrollable)
+    cdkScrollable: CdkScrollable;
```

Comment:
Why do we need to listen to inner scrollable? When we have inner scrollable it looks ugly.
I guess we need to have tab-container already scrollable (not sure if it should be one scrollable for all tabs content or each tab content should be scrollable by its own).

Reply:
removed inner scrollable as it is not needed now.